pyproject.toml

```diff
@@ -0,0 +1,219 @@
+[build-system]
+requires = ["hatchling", "hatch-openzim"]
+build-backend = "hatchling.build"
+
+[project]
+name = "devdocs2zim"
+requires-python = ">=3.12,<3.13"
+description = "Make ZIM files from DevDocs.io"
+readme = "README.md"
+dependencies = [
+    "requests==2.31.0",
+]
+dynamic = ["authors", "classifiers", "keywords", "license", "version", "urls"]
+
```

Comment:
I just realized that you miss a section to configure and trigger the automatic population of authors, classifiers, keywords, license and urls metadata with hatch-openzim plugin.

Following configuration is probably OK to populate default metadata for scrapers + add a "devdocs" keyword.
```
[tool.hatch.metadata.hooks.openzim-metadata]
kind = "scraper"
additional-keywords = ["devdocs"]
```

At least this is the "usual" configuration on other scrapers, see e.g.:
- https://github.com/openzim/ted/blob/main/pyproject.toml#L23-L25
- https://github.com/openzim/youtube/blob/main/scraper/pyproject.toml#L27-L29